dev/tasks/tasks.yml

```diff
@@ -1085,6 +1087,22 @@ tasks:
       - plasma-glib-libs-{no_rc_version}-1.el7.x86_64.rpm
       - plasma-libs-{no_rc_version}-1.el7.x86_64.rpm
 
+  ############################## Homebrew Tasks ################################
+
+  homebrew-cpp:
+    ci: travis
+    platform: osx
+    template: homebrew-formulae/travis.osx.yml
+    params:
+      formula: ci/apache-arrow.rb
+
+  homebrew-cpp-autobrew:
+    ci: travis
+    platform: osx
+    template: homebrew-formulae/travis.osx.yml
+    params:
+      formula: r/tools/apache-arrow.rb
```

Comment:
I'd keep it under `dev/tasks/homebrew-formulae` as well. To remain consistent with the rest of the packaging definitions. We can choose later to organize these files per language backend, but that can be a follow-up.

Reply:
Mildly inconvenient to do so, but it's done now.